README.md

```diff
@@ -131,25 +131,27 @@ preview.show(fileId, accessToken, {
+| downloadWM                       | false         | Whether the download of a watermarked file should be watermarked. This option does not affect non-watermarked files. If true, users will be able to download watermarked versions of supported file types as long as they have `can_preview` permissions (any collaboration role except for `Uploader`).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| viewers                          | {}            | Maps options to a valid viewer type **Use these at your own risk**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I would remove this "use at your own risk" and leave it on the `disableFontFace` option, instead.